lizrd/generate_raport_model_fits.py

```diff
@@ -0,0 +1,76 @@
+import yaml
+import argparse
+import json
+import pandas as pd
```

Comment:
I don't know how to mark it, but a file rename would be good here: maybe `generate_gpu_usage_report`?